src/post_processing.jl

```diff
@@ -535,6 +521,90 @@ function write_results(
     return Dict("bus_results" => bus_df, "flow_results" => branch_df)
 end
 
+"""
+Returns a dictionary containing the DC power flow results. Each key conresponds
+to the name of the considered time periods, storing a DataFrame with the OPF
+results.
+
+# Arguments:
+- `data::Union{PTDFPowerFlowData, vPTDFPowerFlowData, ABAPowerFlowData}`:
+        PowerFlowData strcuture containing power flows and bus angles.
+- `sys::PSY.System`:
+        container storing the systam information.
+"""
+function write_results(
+    data::Union{PTDFPowerFlowData, vPTDFPowerFlowData, ABAPowerFlowData},
+    sys::PSY.System,
+)
+    @info("Voltages are exported in pu. Powers are exported in MW/MVAr.")
+
+    ### non time-dependent variables
+
+    # get bus and branches
+    branches, buses = _get_branches_buses(data)
+
+    # get branches from/to buses
+    from_bus = Vector{Int}(undef, length(branches))
+    to_bus = Vector{Int}(undef, length(branches))
+    for (i, branch) in enumerate(branches)
+        br = PSY.get_component(PSY.ACBranch, sys, branch)
+        from_bus[i] = PSY.get_number(PSY.get_arc(br).from)
+        to_bus[i] = PSY.get_number(PSY.get_arc(br).to)
+    end
+
+    # if length(data.timestep_map) == 1
```

Comment:
why is this commented out?

Reply:
I forgot the remove that. Now it is done.